Classes/Event/EventMetadata.php

```diff
@@ -11,46 +11,28 @@
  * source code.
  */
 
-use Neos\Cqrs\Domain\Timestamp;
-use TYPO3\Flow\Annotations as Flow;
-
 /**
- * The MessageMetadata is a container for arbitrary metadata for Commands and Events.
+ * The EventMetadata is a container for arbitrary metadata for Events.
  * This class is immutable.
  *
  * @api
  */
-class MessageMetadata
+class EventMetadata
 {
-    /**
-     * @var \DateTimeImmutable
-     */
-    protected $timestamp;
+    const VERSION = 'neos.cqrs:version';
+    const TIMESTAMP = 'neos.cqrs:timestamp';
```

Comment:
I added the constants from the otherwise empty `Metadata` class in order to have these two property names predefined.
Still I think the metadata should just be serialized as a simple key/value array (with namespaced keys), like:
```json
{
  "neos.cqrs:version": 123,
  "your.package:someCustomProperty": "foo"
}
```